specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2022-02-01/kusto.json

```diff
@@ -180,12 +180,24 @@
           },
           "201": {
             "description": "Successfully updated the cluster.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
```

Comment:
If it is `x-ms-long-running-operation`, I don't think you need to explicitly specify `Azure-AsyncOperation` as it is well-known header.